src/main/java/com/beerjournal/breweriana/item/ItemService.java

```diff
@@ -34,4 +34,21 @@ ItemDto addItem(String ownerId, ItemDto itemDto) {
         return itemDto;
     }
 
+    ItemDto deleteItem(String ownerId, String itemId) {
```

Comment:
I saw that you implemented it by SecurityUtils, but this approach doesn't look too elegant to me. Don't you know if there isn't any better, more 'declarative' way? E.g. with security configuration?